boards/mulle/board.c

```diff
@@ -61,6 +61,10 @@ static mtd_spi_nor_t mulle_nor_dev = {
     .addr_width = 3,
     .mode = SPI_MODE_3,
     .clk = SPI_CLK_10MHZ,
+    .wait_chip_erase = 16 * US_PER_SEC,
```

Comment:
Isn't this defaulting to `int` as result for the calculation? (change to `16LU * ...`)?

Reply:
Changed all values in the struct for consistency